INTERPRETING.md

```diff
@@ -135,9 +135,11 @@ This must precede any additional text modifications described by test metadata.
 ### Modules
 
 Test262 includes tests for ECMAScript 2015 module code, denoted by the "module"
-metadata flag. Files bearing a name ending in `_FIXTURE.js` should not be
-interpreted as standalone tests; they are intended to be referenced by test
-files.
+metadata flag. Files bearing a name ending in `_FIXTURE.js`:
+
+- **MUST** not be interpreted as standalone tests; they are intended to be referenced by test files.
+- **MUST** not refer to, or make use of any [Test262-Defined Bindings](#test262-defined-bindings) in any way. 
+- **MUST** not refer to, or make use of any [Host-Defined Functions](#host-defined-functions) in any way. 
```

Comment:
thanks for replacing should with must. That's accurate.

What do you think of highlighting `**MUST NOT**` instead? It sounds more clear.

Reply:
Done!